formapi/api.py

```diff
@@ -213,7 +204,7 @@ def dispatch(self, request, *args, **kwargs):
         # Authorize request
         if access_granted:
             self.log.info("Access Granted %s", self.get_request_params())
-            return super().dispatch(request, *args, **kwargs)
+            return super(API, self).dispatch(request, *args, **kwargs)
```

Comment:
Why should we add this back?

Reply:
We should not, it's a mistake from my merge